e2e-tests/commands/insurance/assert-alternative-currency-form.js

```diff
@@ -85,6 +106,73 @@ const assertAlternativeCurrencyForm = ({
     const expectedValue = `${AED.name} (${AED.isoCode})`;
     checkAutocompleteInput.allowsUserToRemoveCountryAndSearchAgain(countryInput.field(ALTERNATIVE_CURRENCY_FIELD_ID), DZA.NAME, AED.name, expectedValue);
   },
+  rendersAlternativeCurrencyValidationError: () => {
+    const option5 = whichRadio(FIELD_ID, ALTERNATIVE_CURRENCY_FIELD_ID);
+
+    option5.input().click();
+    cy.clickSubmitButton();
+
+    cy.checkText(
+      partials.errorSummaryListItems().first(),
+      ERRORS[ALTERNATIVE_CURRENCY_FIELD_ID].IS_EMPTY,
+    );
+
+    cy.checkText(
+      field(ALTERNATIVE_CURRENCY_FIELD_ID).errorMessage(),
+      `Error: ${ERRORS[ALTERNATIVE_CURRENCY_FIELD_ID].IS_EMPTY}`,
+    );
```

Comment:
`cy.submitAndAssertFieldErrors`?

Reply:
will not work because there are different fieldIds.  Everywhere where we have different fieldIds for the input needs it this way round